Index: vendor/freemius/php-sdk/freemius/Freemius.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- vendor/freemius/php-sdk/freemius/Freemius.php	(date 1592846584234)
+++ vendor/freemius/php-sdk/freemius/Freemius.php	(date 1592846584234)
@@ -356,7 +356,7 @@
                 $body .=
                     ('--' . $pBoundary . PHP_EOL) .
                     ("Content-Disposition: form-data; name=\"{$name}\"; filename=\"{$filename}\"" . PHP_EOL) .
-                    ('Content-Type: ' . $this->GetMimeContentType($filename) . PHP_EOL) .
+                    ('Content-Type: ' . $this->GetMimeContentType(strtolower($filename)) . PHP_EOL) .
                     PHP_EOL .
                     (file_get_contents($file_path) . PHP_EOL);
             }
